packages/integrations/image/src/lib/get-image.ts

```diff
@@ -1,14 +1,15 @@
 import slash from 'slash';
-import { ROUTE_PATTERN } from './constants.js';
+import { ROUTE_PATTERN } from '../constants.js';
+import sharp from '../loaders/sharp.js';
```

Comment:
TEMP: avoiding another globalThis here by depending on the sharp import

When we add squoosh, this will import from a helper that checks if sharp is installed first